src/stores/messages/MessagesStore.ts

```diff
@@ -128,6 +135,88 @@ export default class MessagesStore {
 		return [timestampTo - 15 * 1000, timestampTo + 15 * 1000];
 	}
 
+	public includes(message: EventMessage) {
+		return (
+			this.exportMessages.filter(exportMessage => exportMessage.messageId === message.messageId)
```

Comment:
i think we can even check it by reference and not by id, and replace filter with includes